### DISCUSSION TITLE
Does Mariadb included in this docker image?

### DISCUSSION THREAD

**Discussion by x-Felix** [librenms/docker#366] [Q&A] [answered]
I am trying to get some clarification of usage of this docker image. 
In the Readme, Features section, it says MariaDB image as database instance. And Dockerfile shows mariadb-dev is installed. 
Without define db Environment variables, container complains no database defined. 
How can I use builtin mariadb? Or I have to spin up another mariadb container? Thanks.

**Comment by ninkaninus** [comment]
Try asking it here:
https://t.libren.ms/discord

**Comment by crazy-max** [comment, marked as answer]
@x-Felix 

> In the Readme, Features section, it says MariaDB image as database instance.

It is used as a dedicated container through our [compose example](https://github.com/librenms/docker/blob/4be8d30631a7721a58e6a7527775f395321de8ff/examples/compose/docker-compose.yml#L5-L21) like RRDCached, Memcached, Redis and MSMTPD.
